packages/retail-ui-extensions/src/extension-api/smartgrid-api/smartgrid-api.ts

```diff
@@ -1,10 +1,7 @@
-import type {Modal} from '../types';
-
 export interface SmartGridApiContent {
-  /** Presents a new modal on top of the Smart Grid.
-   * @param modal this interface contains the information about the path and any initial parameters.
+  /** Presents the `pos.home.modal.render` extension point on top of the Smart Grid.
    */
-  presentModal(modal: Modal): void;
+  presentModal(): void;
```

Comment:
Two notes about what this new approach misses compared to the old one: 

1. Sometimes the user might want to start with a different screen depending on the tile state. Perhaps we should have a `destination` here.
2. Sending `params` here could also be useful in case a `Tile` needs to share some state with the `Modal`.

Reply:
Hmmmm yeah fair enough

Reply:
@heltisace `navigationApi`'s `navigate` method takes in the same parameters as well. It takes a screen/modal name and a params. IMO we should either share the destination type with it OR, we don't create a type, we just have a `destinationName: string, params: any`. I'm leaning more on just having regular primitive function parameters, rather than a typed object. It's just two parameters. What do you think?